lib/triggers/botCommandTrigger.js

```diff
@@ -39,10 +41,22 @@ BotCommandTrigger.prototype._respondToChatMessage = function(roomId, chatterId,
 
 BotCommandTrigger.prototype._triggerOnMatch = function(fromId,toId,message) {
 	if (this._messageTriggers(message) && this.options.callback) {
-		if(this.options.callback.length===1) {
-			this.options.callback(this.chatBot);
+		if ('function' === typeof this.options.callback) {
+			if (this.options.callback.length === 1) { // check number of args
+				this.options.callback(this.chatBot);
+			} else {
+				this.options.callback(this.chatBot, {toId: toId, fromId: fromId, message: message});
+			}
 		} else {
-			this.options.callback(this.chatBot, {toId:toId,fromId:fromId,message:message});
+			if ('string' === typeof this.options.callback) {
+				this.options.callback = [this.options.callback];
+			}
+			var callbackArgs = this.options.callback.slice();
+			var callback = callbackArgs.shift();
+			if (typeof callback === "string" && callback.substring(0, 1) != "_" && callback in this.chatBot
```

Comment:
Not sure if I'm reading this right...it allows you to specify a function of chatBot and call it as a string? Seems interesting.